NEWS

```diff
@@ -1,3 +1,10 @@
+etcd v3.0.5 (2016-08-19)
+
+  - If etcd is using TLS without custom certificate authority, the discovery
```

Comment:
SRV records (e.g., infra1.example.com) must match the discovery domain (i.e., example.com) when using the default certificate authority.